WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs

```diff
@@ -1061,6 +1061,14 @@ public IEnumerable<Alice> GetAlicesBy(AliceState state)
 			}
 		}
 
+		public IEnumerable<Alice> GetAlicesBy(BitcoinAddress changeOutputAddress)
```

Comment:
Sorry for the late response. Evaluating if Length is not zero is basically an Any query. Proposal: Rename this to IsChangeOutputRegistered(), use Any, and return bool.

Reply:
FIxed